packages/prettier/source/prettier.ts

```diff
@@ -0,0 +1,25 @@
+import type { Options } from "prettier";
+
+// @see https://prettier.io/docs/en/options.html
+const config: Options = {
```

Comment:
@acwo 

I made sure its consistent with the `.editorconfig` from the #1.

**There's also a question we need to answer and decide together.**

Should we create a promised(?) democratic vote either in this Pull Request on how we want our coding format guidelines to be set and maintained?


P.S. Of course just in the currently active project for starters.

Reply:
> Should we create a promised(?) democratic vote in this Pull Request on how we want our coding format guidelines to be set and maintained?
- I don't think so -  we've had this kind of conversation before, and all of our current configurations are the result of common agreements.